applications/zencode-for-substrate.md

```diff
@@ -0,0 +1,129 @@
+# Open Grant Proposal
+
+* **Project Name:** Zencode for Substrate :left_right_arrow:
+* **Team Name:** CryptoArena
+* **DAI Payment Address:** `0x8214864d37944082322745b1639828F680f02124`
+
+## Project Overview :page_facing_up: 
+This application is an RFP for $30,000 in DAI.
```

Comment:
Do you mean, this application is in response to a Request For Proposals? If so, could you add a link? Or what do you mean by RFP?

Reply:
Hi @semuelle, I read somewhere in the instructions we had to specify if it was a request for proposal - by which we intend a document to solicit a request, from you to us, to build the proposed milestone. 

Could be a misunderstanding on my part if you mean something else.

Reply:
I see. Well, since you already know what you want built and you want to implement it yourself, this would be considered a grant application. RFPs can indeed by written by anyone, but they are intended to attract developers, who can then submit grant applications in reference to an RFP. So, "proposals" are grant applications, and RFPs are "invitations for grant applications" for a specific topic. RFPs lack the most specific information, such as the name of a/the team implementing it (see our [RFP template](https://github.com/w3f/General-Grants-Program/blob/master/rfp-proposal/suggestion-template.md)).

Reply:
Understood. Thanks for the disambiguation, we'll amend it to say its a grant application

Reply:
@semuelle and @CryptoArenaCEO, I've updated the application to indicate that it is a grant application instead of an RFP.